SPECS/mariner-rpm-macros/macros.python3

```diff
@@ -50,6 +50,11 @@
   rm -rfv %{buildroot}%{_bindir}/__pycache__
 }
 
+%py3_check() %{expand:\\\
```

Comment:
I get the changes in `protobuf.spec`, but why are we adding this new macro? Could you mention and explain the need for it in the PR's summary?

Reply:
In earlier commits of the PR I used the macro to call and run python tests, it worked but produced failures in tests. Then I found out that protobuf uses different tool called `tox` for their tests. We have it as an extended package. I could not get `tox` to build, I tried updating it too and still no luck. But my understand is that this is how you generally run python unit tests, so I kept the macro in case we want to use it in future for other packages

Reply:
I'm generally against adding this macro.

The `python3 setup.py test` entrypoint is [deprecated upstream](https://github.com/pypa/setuptools/issues/1684). The macros in this file are... generally deprecated in favor of the ones in `pyproject-rpm-macros` for that reason. I'd rather move towards the macros there instead of adding yet another deprecated macro that will need to be cleaned up in 3.0.

Reply:
> I'm against adding this macro. I'm not against locally defining it in the `protobuf` spec, though.
> 
> The `python3 setup.py test` entrypoint is [deprecated upstream](https://github.com/pypa/setuptools/issues/1684). The macros in this file are... generally deprecated in favor of the ones in `pyproject-rpm-macros` for that reason. I'd rather move towards the macros there instead of adding yet another deprecated macro that will need to be cleaned up in 3.0.

Oh, I did not know it was deprecated. I will remove the addition of it then,

Reply:
Thanks, I appreciate the effort you put in to adding it though!